### PR TITLE
Test framework

### PR DESCRIPTION
**Pull request by baude**
A series of small commits that fix or enhance integration testing.

### CODE REVIEW

File: test.sh [hunk 1]
```diff
@@ -113,15 +113,16 @@ make_docker_images () {
         if [[ -d "${df}.d" ]]; then
             cp -r "${df}.d" "${WORK_DIR}/${BASE_NAME}.d"
         fi
-
+	SECONDS=0
         ${DOCKER} build -t ${iname} -f ${df_cp} ${WORK_DIR} >> ${LOG}
+	DURATION=$SECONDS
```
Comment (aweiteka) [inline test.sh]: When is DURATION ever updated? Won't it always be 0?

File: Atomic/storage.py [hunk 2]
```diff
@@ -63,6 +63,9 @@ def cli(subparser):
                          help=_("remove all unused block devices from storage pool"))
     modifyp.add_argument('--driver', dest="driver", default=None, help='The storage backend driver', choices=['devicemapper', 'overlay', 'overlay2'])
     modifyp.add_argument('--vgroup', dest="vgroup", default=None, help='The storage volume group')
+    modifyp.add_argument("--graph", dest="graph",
+                        default=default_docker_lib(),
+                        help=_("Root of the Docker runtime (Default: %s)" % default_docker_lib()))
```
Comment (aweiteka) [inline Atomic/storage.py]: Is this related to the test work? Looks like the opt was documented but never implemented?

File: test.sh (hunk 1 above)
Comment (baude) [inline test.sh]: No it is a special bash function.  Seems to work:

```
	Built   : atomic-test-1 in 16 seconds
	Built   : atomic-test-3 in 9 seconds
	Built   : atomic-test-4 in 7 seconds
	Built   : atomic-test-5 in 8 seconds
	Built   : atomic-test-6 in 17 seconds
	Built   : atomic-test-secret in 7 seconds
	Built   : atomic-test-system in 22 seconds
```

File: Atomic/storage.py (hunk 2 above)
Comment (baude) [inline Atomic/storage.py]: Right, I agree.  Indeed it actually causes a test failure.

**Comment by baude** [comment]
@rh-atomic-bot r+

**Comment by baude** [comment]
@rh-atomic-bot r+ eed4861
